starter-code/css/main.css

```diff
@@ -6,3 +6,176 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+background-color: #FFF;
+color : 1A1A1A;
+font-style: Proxima Nova Regular;
+max-width: 100%;
+}
+
+.nav-bar-format {
+  position: fixed;
+  background-color: white;
+  margin: 0 0 0 0;
+  padding: 0;
+  list-style: none;
+  width: 100%;
+  height: 85px;
+  font-size: 22px;
+  text-align: center;
+  z-index: 1;
+    }
+
+.nav-flex-align {
+  display: flex;
+  justify-content: space-around;
+}
+
+.nav-menu-flex-align {
+  display: flex;
+  justify-content: flex-end;
+  position: relative;
+  right: -90px;
+  top: 15px;
+}
+
+li {
+  text-decoration: none;
+  list-style-type: none;
+  width: 80px;
+  margin: 0 10px 0 10px;
+  padding: 0;
+  background-color: #FFF
+  }
+
+.nav-image {
+  position: relative;
+  top: 5px;
+  left:5px;
+  display: block;
+  width: 200px;
+  top: 15px;
+  left: -120px;
+}
+
+main {
+  background-image: url("../images/landing.jpg");
+  background-position:center center;
+  background-size: 100%;
+  position : relative;
+  top: 90px;
+  border-style: solid;
+  background-repeat:no-repeat;
+  width: 100%;
+  height: 800px;
+  color: white;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  resize: vertical;
+  overflow: auto;
+}
+
+main h1 {
+  font-size: 80px
+}
+
+main h2 {
+  font-size: 25px
+}
+
+.whatsonstyle {
+  position: relative;
+  top: 90px;
+  border-style: none none solid none;
+  border-color: #00B172;
+  text-align: center;
+  font-size: 30px;
+  width: 250px;
+  height:40px;
+  padding: 0px;
+  margin: 0 auto;
+}
+
+.iconspos {
+  display: flex;
+  justify-content: space-around;
+  position: relative;
+  top: 200px;
+}
+
+.iconspos div {
+  width: 200px;
+  height: 100px;
+  text-align: center;
+}
+
+.subiconspos {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.iconspos img {
+height: auto;
+width: auto;
+max-width: 100px;
+max-height: 100px;
+}
+
+h5 {
+  color: #00B172;
+  font size: bold;
+  font-size: 18px;
+}
+
+.last-section-pos {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  top:350px;
+  background-color: #00B172;
+  margin: 10px;
+  height: 550px;
+}
+
+h6, h7{
+  font-size: 18px;
+  color: #FFF;
+}
+
+.last-section-pos p {
+  font-size: 15px;
+  color: #FFF;
+}
+
+.first-box-size {
+  width: 400px;
+  height: 500px;
+  border-style: solid;
+}
+
+.middle-box-size {
+  width: 100px;
+  height: 100px;
+  border-style: solid;
+
```

Comment:
You didn't close the block here. It will silently fail so be very diligent about this.